python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -144,18 +168,66 @@ def convert_in_position(xyin, inr, c, cent):
     elif c.mode == 'mcs_pfi_asrd':
         xyconv = pixel_to_mm(xyin, inr, cent,
                              pix=DCoeff.mcspixel_asrd, invx=-1., invy=1.)
+    elif (c.mode == 'sky_pfi') or (c.mode == 'sky_pfi_hsc'):
+        # Set Observation Site (Subaru)
+        tel = EarthLocation.of_site('Subaru')
+        tel2 = Observer.at_site("Subaru", timezone="US/Hawaii")
+        obs_time = Time(time)
+
+        aref_file = mypath+'data/Refraction_data_635nm.txt'
+        atm_ref = np.loadtxt(aref_file)
+        atm_interp = ipol.splrep(atm_ref[:, 0], atm_ref[:, 1], s=0)
+
+        # Ra-Dec to Az-El (Center)
+        coord_cent = SkyCoord(cent[0], cent[1], unit=u.deg)
+        altaz_cent = coord_cent.transform_to(AltAz(obstime=obs_time,
+                                                   location=tel))
+
+        # Instrument rotator angle
+        paa = tel2.parallactic_angle(obs_time, coord_cent).deg
+        lat = tel2.location.lat.deg
+        dc = coord_cent.dec.deg
+        if dc > lat:
+            inr = paa + pa - 180.
+        else:
+            inr = paa - pa
+
+        az0 = altaz_cent.az.deg
+        el0 = altaz_cent.alt.deg
+        za = 90. - el0
+        eld0 = el0 + ipol.splev(za, atm_interp)/3600.
+        za = za[0]
```

Comment:
I don't know why, but for me the code only runs when I comment out this line. `za` is not an array when I'm running the code; perhaps I have different versions of some Python packages?

Reply:
Well.. looking this again, this line seems strange, but I can run the script with python 3.6 as it is... but I'll look into it.

Reply:
I checked data type and I remember the reason.... `za` becomes array in line 197 to pass scipy `ipol.splev` function in line 198. Then I change it back to float... Anyway these lines are confusing and not so normal.. I'll modify here to seeleaner.